src/qml/pages/node/Shutdown.qml

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../../controls"
+
+Page {
+    background: null
+
+    ColumnLayout {
+        anchors.centerIn: parent
+        width: parent.width
+        spacing: 10
+        Button {
+            Layout.alignment: Qt.AlignCenter
+            Layout.bottomMargin: 20
+            background: null
+            icon.source: "image://images/shutdown"
+            icon.color: Theme.color.neutral9
+            icon.width: 60
+            icon.height: 60
+        }
+        Text {
```

Comment:
this is a good place to just use `Header`:

``` diff
diff --git a/src/qml/pages/node/Shutdown.qml b/src/qml/pages/node/Shutdown.qml
index 8dc95d258..36c7ab2af 100644
--- a/src/qml/pages/node/Shutdown.qml
+++ b/src/qml/pages/node/Shutdown.qml
@@ -23,30 +23,20 @@ Page {
             icon.width: 60
             icon.height: 60
         }
-        Text {
-            Layout.alignment: Qt.AlignCenter
+        Header {
             Layout.fillWidth: true
             Layout.leftMargin: 20
             Layout.rightMargin: 20
-            horizontalAlignment: Text.AlignHCenter
-            text: qsTr("Saving...")
-            font.family: "Inter"
-            font.styleName: "Semi Bold"
-            font.pixelSize: 28
-            color: Theme.color.neutral9
+            header: qsTr("Saving...")
+            headerBold: true
         }
-        Text {
-            Layout.alignment: Qt.AlignCenter
+        Header {
             Layout.fillWidth: true
             Layout.leftMargin: 20
             Layout.rightMargin: 20
-            horizontalAlignment: Text.AlignHCenter
-            text: qsTr("Do not shut down the computer until this is done.")
-            font.family: "Inter"
-            font.styleName: "Regular"
-            font.pixelSize: 18
-            color: Theme.color.neutral7
-            wrapMode: Text.WordWrap
+            header: qsTr("Do not shut down the computer until this is done.")
+            headerSize: 18
+            headerColor: Theme.color.neutral7
         }
     }
 }
```

Reply:
| patch | pr |
| ---- | -- |
| <img width="752" alt="path" src="https://user-images.githubusercontent.com/23396902/220184854-ad17f223-355b-47d9-a8cc-fac240454caa.png"> | <img width="752" alt="master-a" src="https://user-images.githubusercontent.com/23396902/220184871-1b75de0c-d1e5-4388-80da-5b4286dfb67a.png"> |